### ISSUE TITLE
Setup the Workspace does not work.

### ISSUE THREAD

**Issue by alkatar21** [iterative/vscode-dvc#2660]
dvc: 2.30.1
dvc VS-Code extension: v0.5.5

I have installed dvc via pip and installed the extension.
I tried to init a new project with the extension, but currently this is not possible. I tried both ways and all options available. dvc works in VSCode console.
![grafik](https://user-images.githubusercontent.com/61387986/197282909-325a122b-ece1-4541-8bcc-e6297b2d4160.png)
![grafik](https://user-images.githubusercontent.com/61387986/197282940-21497767-8b31-4071-9aab-d01cc63c4aff.png)

The output window shows the following:
```bash
[version: 0.5.5, ...] > /bin/python3 -m dvc root - INITIALIZED
[version: 0.5.5, ...] > /bin/python3 -m dvc root - INITIALIZED
[version: 0.5.5, ...] > /bin/python3 -m dvc --version - INITIALIZED
[version: 0.5.5, ...] > /bin/python3 -m dvc --version - COMPLETED (327ms)
[version: 0.5.5, ...] > /bin/python3 -m dvc root - FAILED with code 253 (515ms)
[31mERROR[39m: you are not inside of a DVC repository (checked up to mount point '/workspaces/<project>')
```

Additional context: https://discord.com/channels/485586884165107732/842220310585147452/1033034387060703333

**Comment by shcheklein** [comment]
thansk @alkatar21 ! we are looking into this.

**Comment by mattseddon** [comment]
The extension needs a DVC project to operate on. However, under these circumstances the sidebar should look like this:

![image](https://user-images.githubusercontent.com/37993418/197296959-e922d5d4-dd82-4e55-8395-f8f390d8d974.png)

I will raise a PR to fix the bug but you should be able to work around the issue by running `dvc init` in your terminal.

**Comment by mattseddon** [comment]
Should be fixed by `0.5.7` 🙏🏻.
